src/polygonFeature.js

```diff
@@ -151,6 +151,32 @@ var polygonFeature = function (arg) {
 
   ////////////////////////////////////////////////////////////////////////////
   /**
+   * Get the style for the stroke of the polygon.  Since polygons can have
+   * holes, the number of stroke lines may not be the same as the number of
+   * polygons.  If the style for a stroke is a function, this calls the
+   * appropriate value for the polygon.  Any style set for a stroke line should
+   * be wrapped in this function.
+   *
+   * @param {(object|function)?} styleValue The polygon's style value used for
+   *    the stroke.  This should be m_this.style(<name of style>) and not
+   *    m_this.style.get(<name of style>), as the result is more efficient if
```

Comment:
Our style function can either be called like `style('strokeColor')`, which returns the value set for strokeColor, or as `style.get('strokeColor')`, which *always* returns a function for strokeColor.  Here, if we know that the value is *not* a function, then we can pass it along to the stroke, whereas if it *is* a function, we have to get the appropriate value for the polygon by calling that function with the polygon's values.